ddtrace/internal/telemetry/writer.py

```diff
@@ -502,6 +500,51 @@ def add_log(self, level, message, stack_trace="", tags=None):
             # Logs are hashed using the message, level, tags, and stack_trace. This should prevent duplicatation.
             self._logs.add(data)
 
+    def add_integration_error_log(self, msg, exc, warning=False):
+        if warning is False:
+            log.debug(msg, exc_info=exc)
+        else:
+            log.warning(msg, exc_info=exc)
+
+        if config.LOG_COLLECTION_ENABLED:
+            stack_trace = self._format_stack_trace(exc)
+            self.add_log(
+                TELEMETRY_LOG_LEVEL.ERROR,
+                msg,
+                tags={"lib_language": "python"},
```

Comment:
Every telemetry payload has a `lib_language` field/http-header. Do we need to duplicate this attribute here?